src/main/java/com/restaurantpicker/RestaurantpickerApplication.java

```diff
@@ -1,7 +1,5 @@
 package com.restaurantpicker;
 
-import java.util.Arrays;
```

Comment:
According to Java best practices, useless code (classes that are never used, etc.) should not be committed to GitHub unless it provides some functionality and both these imports were never used.